operator_ui/src/Private.js

```diff
@@ -68,7 +68,7 @@ class Private extends React.Component {
     let drawerContainer
 
     return (
-      <Grid container style={{width: '100%'}}> {/* Horizontal Scrollbar Fix for MUI Container calc(100% + Npx) width */}
+      <Grid container>
```

Comment:
@rupurt There's also an alternative approach trying `spacing={0}` after removing `style={{width: '100%'}}`. I briefly tested locally and didn't find anything bad with it. I'm generally in favor of fixing things like this without `overflow-x: hidden` unless absolutely necessary. What do you think?

Reply:
@NavyAdmiral works for me. Updated